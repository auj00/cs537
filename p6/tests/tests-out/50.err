Traceback (most recent call last):
  File "/home/yashd/private/CS537_Team/cs537/p6/tests/./readdir-check.py", line 8, in <module>
    foundfiles = os.listdir("mnt")
OSError: [Errno 38] Function not implemented: 'mnt'
