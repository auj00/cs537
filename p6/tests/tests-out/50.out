Correct
readdir files don't match expectation
